src/Sylius/Bundle/AdminBundle/Resources/views/Customer/_form.html.twig

```diff
@@ -17,20 +17,22 @@
         </div>
     </div>
     <div class="column">
-        <div class="ui segment">
-            <h4 class="ui dividing header">{{ 'sylius.ui.account_credentials'|trans }}</h4>
-            <div class="field">
-                <div class="ui toggle checkbox">
-                    <input type="checkbox" id="sylius_customer_create_user" {% if customer.user is not empty %}checked="checked"{% endif %}>
-                    <label for="sylius_customer_create_user">{{ 'sylius.ui.customer_can_login_to_the_store'|trans }}?</label>
+        {% if customer.user is empty %}
+            <div class="ui segment">
+                <h4 class="ui dividing header">{{ 'sylius.ui.account_credentials'|trans }}</h4>
+                <div class="field">
+                    <div class="ui toggle checkbox">
+                        <input type="checkbox" id="sylius_customer_create_user">
+                        <label for="sylius_customer_create_user">{{ 'sylius.ui.customer_can_login_to_the_store'|trans }}?</label>
+                    </div>
                 </div>
+                {% if form.user is defined %}
+                    <div id="user-form" {% if form.user.vars.data.id is not defined %} style="display: none" {% endif %}>
+                        {{ form_row(form.user.plainPassword) }}
+                        {{ form_row(form.user.enabled) }}
+                    </div>
+                {% endif %}
```

Comment:
When the customer has user (registered account), we still want to display user form, but the only thing we want to hide is the checkbox :)